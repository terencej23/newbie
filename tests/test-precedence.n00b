define function main with no params
	set x to 2 + 3 * 4

	print(x)

	set x2 to (2 + 3) * 4

	print(x2)

	set y to 9 - 6 / 3

	print(y)

	set y2 to (9 - 6) / 3

	print(y2)
