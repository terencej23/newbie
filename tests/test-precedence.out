14
20
7
1